Documentation/devel/pod-lifecycle.md

```diff
@@ -83,22 +95,44 @@ All directories found in `$var/run` are tested for exited status by trying to ac
+Marked exited pods dwell in the `$var/exited-garbage` directory for a grace period during which their status may continue to be queried by `rkt status`.
+The rename from `$var/run/$uuid` to `$var/exited-garbage/$uuid` serves in part to keep marked pods from cluttering the `$var/run` directory during their respective dwell periods.
 
 ### Pass 2: sweep
 
-A side-effect of the rename operation responsible for moving a pod from `$var/run` to `$var/exited-garbage` is an update to the pod directory's change time.  The sweep operation takes advantage of this in honoring the necessary grace period before discarding exited pods.  This grace period currently defaults to 30 minutes, and may be explicitly specified using the `--grace-period=duration` flag with `rkt gc`.  Note that this grace period begins from the time a pod was marked by `rkt gc`, not when the pod exited.  A pod becomes eligible for marking upon exit, but will not become marked until a subsequent `rkt gc` is performed.
-
-The change times of all directories found in `$var/exited-garbage` are compared against the current time.  Directories having sufficiently old change times are locked exclusively and cleaned up.  If a lock acquisition fails, the directory is skipped.  Failed exclusive lock acquisitions may occur if the garbage pod is currently being accessed via `rkt status`, or deleted by a concurrent `rkt gc`, for example.  The skipped pods will be revisited on a subsequent `rkt gc` invocation's sweep pass.  During the cleanup, the pod's stage1 gc entry point is first executed.  This gives the stage1 a chance to clean up anything related to the environment shared between containers. The default stage1 uses the gc entrypoint to clean up the private networking artifacts.  After the completion of the gc entrypoint, the pod directory is recursively deleted.
+A side-effect of the rename operation responsible for moving a pod from `$var/run` to `$var/exited-garbage` is an update to the pod directory's change time.
+The sweep operation takes advantage of this in honoring the necessary grace period before discarding exited pods.
+This grace period currently defaults to 30 minutes, and may be explicitly specified using the `--grace-period=duration` flag with `rkt gc`.
+Note that this grace period begins from the time a pod was marked by `rkt gc`, not when the pod exited.
+A pod becomes eligible for marking upon exit, but will not become marked until a subsequent `rkt gc` is performed.
+
+The change times of all directories found in `$var/exited-garbage` are compared against the current time.
+Directories having sufficiently old change times are locked exclusively and cleaned up.
+If a lock acquisition fails, the directory is skipped.
+Failed exclusive lock acquisitions may occur if the garbage pod is currently being accessed via `rkt status`, or deleted by a concurrent `rkt gc`, for example.
```

Comment:
`rkt gc` may fail to acquire an exclusive lock if the pod to be collected is currently being accessed, by `rkt status` or another `rkt gc`, for example.